src/amoc.app.src

```diff
@@ -29,10 +28,7 @@
                 []},
                {[amoc, users],
                 {function, ets, info, [amoc_users], proplist, [size]},
-                []},
-               {[amoc, times, connection], histogram, []},
-               {[amoc, counters, connections], spiral, []},
-               {[amoc, counters, connection_failures], spiral, []}
```

Comment:
These metrics are used by existing scenarios (in amoc-arsenal-xmpp). Are you going to move creation and subscription of these metrics to `amoc-arsenal-xmpp` in a linked PR? If no I'd rather leave them here.

Reply:
yeap,  this one must be moved to amoc-arsenal-xmpp